security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,25 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-5-0"]
+== OpenShift Compliance Operator 1.5.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.5.0:
+
+* link:https://access.redhat.com/errata/RHBA-2024:3533[RHBA-2024:3533 - OpenShift Compliance Operator 1.5.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-5-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* As of this release, the Compliance Operator provides a unique profile ID for easier programmatic use.  (link:https://issues.redhat.com/browse/CMP-2450[*CMP-2450*])
+
+* With this release, the Compliance Operator provides PCI-DSS-NODE scans and CIS-NODE profile scans for ROSA HCP. The Compliance Operator loads only Node profiles when running on ROSA HCP because is a Red Hat managed platform which restricts access to the control plane, making Platform profiles out of scope for the operator.(link:https://issues.redhat.com/browse/CMP-2581[*CMP-2581*])
```

Comment:
We might be able to get rid of "the Compliance Operator provides PCI-DSS-NODE scans and CIS-NODE profile scans for ROSA HCP. " since it's somewhat redundant with the following sentence.

Reply:
Done, thank you!